rush.json

```diff
@@ -3,7 +3,7 @@
   "rushVersion": "5.83.1",
   "pnpmVersion": "7.13.0",
   "nodeSupportedVersionRange": ">=16.13.0 <17.0.0 || >=18.14.0 <19.0.0",
-  "ensureConsistentVersions": true,
+  "ensureConsistentVersions": false,
```

Comment:
Allows `fwoa-tools` to have a different package version of `axios` from the other projects in the repo